msu/utils/math.nut

```diff
@@ -28,4 +28,18 @@
 		::MSU.requireFloat(_min, _max);
 		return _min + (::Math.rand(0, 2147483647) / 2147483647.0) * (_max - _min);
 	}
+
+	function round( _num, _multiple )
+	{
+		if (_multiple <= 0)
+		{
+			::logError("_multiple must be greater than 0");
+			throw ::MSU.Exception.InvalidValue(_multiple);
+		}
+
+		local num = fabs(_num);
+	    local rem = num % _multiple;
+	    local ret = rem < _multiple * 0.5 ? num - rem : num + _multiple - rem;
+	    return _num < 0 ? -ret : ret;
```

Comment:
Seems to be an indentation issue here, probably mixed spaces/tabs

Reply:
Opps didnt mean to resolve

Reply:
Thanks, fixed.